web/packages/teleport/src/AuthConnectors/AuthConnectorTile.tsx

```diff
@@ -108,14 +110,42 @@ export function AuthConnectorTile({
             Set Up <ArrowRight size="small" ml={2} />
           </ButtonSecondary>
         )}
-        {!isPlaceholder && !!onEdit && !!onDelete && (
-          <MenuIcon
-            buttonIconProps={{ style: { borderRadius: `${theme.radii[2]}px` } }}
-          >
-            <MenuItem onClick={onClickEdit}>Edit</MenuItem>
-            <MenuItem onClick={onClickDelete}>Delete</MenuItem>
-          </MenuIcon>
-        )}
+        {!isPlaceholder &&
```

Comment:
Add test for this whole block to ensure the items/buttons appear as expected? Such logic that deals with multiple states are prone to bug/regression.